src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,30 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Current Summary](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+We are looking for people to test `workspace-inheritance` and provide feedback about it.
+
+We would like to know
+* If there were any pain points in porting your project
+* Any errors or bugs that you found in testing
+* Performance concerns
+* Gaps in documentation
+
+Anything that you find please let us know by filing an issue or commenting on
+the [tracking issue](https://github.com/rust-lang/cargo/issues/8415).
```

Comment:
I'd swap these to avoid discussions-via-issues.

Please provide feedback on the tracking issue or create an Issue for any bugs encountered.